crates/ruff_wasm/src/lib.rs

```diff
@@ -161,39 +155,7 @@ impl Workspace {
             task_tags: None,
             typing_modules: None,
             unfixable: None,
-            // Use default options for all plugins.
-            flake8_annotations: Some(flake8_annotations::settings::Settings::default().into()),
-            flake8_bandit: Some(flake8_bandit::settings::Settings::default().into()),
-            flake8_bugbear: Some(flake8_bugbear::settings::Settings::default().into()),
-            flake8_builtins: Some(flake8_builtins::settings::Settings::default().into()),
-            flake8_comprehensions: Some(
-                flake8_comprehensions::settings::Settings::default().into(),
-            ),
-            flake8_copyright: Some(flake8_copyright::settings::Settings::default().into()),
-            flake8_errmsg: Some(flake8_errmsg::settings::Settings::default().into()),
-            flake8_gettext: Some(flake8_gettext::settings::Settings::default().into()),
-            flake8_implicit_str_concat: Some(
-                flake8_implicit_str_concat::settings::Settings::default().into(),
-            ),
-            flake8_import_conventions: Some(
-                flake8_import_conventions::settings::Settings::default().into(),
-            ),
-            flake8_pytest_style: Some(flake8_pytest_style::settings::Settings::default().into()),
-            flake8_quotes: Some(flake8_quotes::settings::Settings::default().into()),
-            flake8_self: Some(flake8_self::settings::Settings::default().into()),
-            flake8_tidy_imports: Some(flake8_tidy_imports::settings::Settings::default().into()),
-            flake8_type_checking: Some(flake8_type_checking::settings::Settings::default().into()),
-            flake8_unused_arguments: Some(
-                flake8_unused_arguments::settings::Settings::default().into(),
-            ),
-            isort: Some(isort::settings::Settings::default().into()),
-            mccabe: Some(mccabe::settings::Settings::default().into()),
-            pep8_naming: Some(pep8_naming::settings::Settings::default().into()),
-            pycodestyle: Some(pycodestyle::settings::Settings::default().into()),
-            pydocstyle: Some(pydocstyle::settings::Settings::default().into()),
-            pyflakes: Some(pyflakes::settings::Settings::default().into()),
-            pylint: Some(pylint::settings::Settings::default().into()),
-            pyupgrade: Some(pyupgrade::settings::Settings::default().into()),
+            ..Options::default()
```

Comment:
@charliermarsh is this the same? Not using `From<Settings> for Options` here allows us to remove a ton of code (and going from settings to options is rather odd)

Reply:
Hmm... I _think_ this change will have the effect that all of these settings will be hidden in the Settings pane by default, rather than showing their defaults (since they will default to `None` rather than `Some(pyupgrade::settings::Options::default())`).

Reply:
I did not verify this myself, and that might be worth it anyway to be honest.

Reply:
I cleared my local storage and ran the playground locally. It is giving me the same configuration as on production